static/scripts.js

```diff
@@ -1604,10 +1604,18 @@
             calculateThumbnailLayout();
             loadLazyStuff();
 
-            localStorage['home-tab'] = paneId;
+            try {
+                localStorage['home-tab'] = paneId;
+            } catch (error) {}
```

Comment:
Should these be console logged or is there no good point?

Reply:
Sure, that could help find a problem if a change made it throw in a different way.